cmd/algod/main.go

```diff
@@ -282,6 +256,43 @@ func main() {
 	}
 	fmt.Fprintf(os.Stdout, "Deadlock detection is set to: %s (Default state is '%s')\n", deadlockState, config.DefaultDeadlock)
 
+	if log.GetTelemetryEnabled() {
+		// Make a copy of config and reset DNSBootstrapID in case it was disabled.
+		cfgCopy := cfg
+		cfgCopy.DNSBootstrapID = telemetryDNSBootstrapID
+
+		// Periodically check SRV records for new telemetry URI
+		go srvUpdaterLoop(1*time.Minute, cfgCopy, s.Genesis.Network, log)
```

Comment:
I'm not opposing to periodically checking the SRV records, but why not to do that in the network library itself ?
This function `func (wn *WebsocketNetwork) meshThread()` is doing exactly that type of work.
Just add the time-check and make the SRV lookup there.

This code doesn't have the shutdown process. Doing it at the network library save you from implementing that "responsibility".

Reply:
I had it implemented this way originally, but Brian preferred keeping it out of the networking code. In either implementation `algoh` doesn't have a `meshThread()` to leverage so it will have to do something like this.

cc: @algobolson

Reply:
So, if you go on that path, than create a trivial service that you'll be able to shutdown. you might want to create a single service for both this as well as for the heartbeat ticker.

Reply:
I tried to put this function in one of our utils/tools packages, but there it was pulling in just the right dependencies to create import cycles. Maybe a new package, I'll take another stab at it and find a better spot for a service.